src/blockdev.rs

```diff
@@ -1,10 +1,23 @@
 use camino::Utf8Path;
 use std::path::Path;
+use std::sync::OnceLock;
 
 use anyhow::{bail, Context, Result};
 use bootc_blockdev::PartitionTable;
 use fn_error_context::context;
 
+static PARENT_DEVICES: OnceLock<Vec<String>> = OnceLock::new();
+
+// Initialize once
+pub fn init_parent_devices(strings: Vec<String>) {
+    PARENT_DEVICES.set(strings).expect("Already initialized");
+}
```

Comment:
This isn't the usual pattern for this. It's better to keep the `static` scoped inside a particular function. Here's an example
https://github.com/bootc-dev/bootc/blob/93b22f4dbc2d54f7cca7c1df3ee59fcdec0b2cf1/ostree-ext/src/globals.rs#L22

And notably this uses `get_or_init` which ensures initialize-at-most-once semantics, which avoids the `expect`.